css/styles.css

```diff
@@ -0,0 +1,212 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+* img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+footer a {
+  text-decoration: none;
+  font-size: 15px;
+}
+
+.nav3 a {
+  color: grey;
+}
+
+.nav3 a:nth-child(1) {
+  text-decoration: none;
+  color: black;
+}
+
+.links_hover:hover {
+  text-decoration: none;
+  color: red;
+}
+
+.white {
+  color: white;
+}
+
+.nav1 {
+  background-color: #ff0500;
+  z-index: 1000;
+}
+
+.z-ind {
+  z-index: 100;
+}
+
+.subscribe {
+  background-color: #ffe400;
+  border-radius: 20px;
+}
+
+.subs {
+  width: 40%;
+}
+
+.half-size {
+  width: 25%;
+  height: 25%;
+}
+
+.opinion img {
+  height: 50%;
+  width: auto;
+}
+
+.stick img {
+  width: 50%;
+  height: auto;
+}
+
+.piggy-cover {
+  background-color: #ff0500;
+  color: white;
+}
+
+.color-red {
+  color: #ff0500;
+}
+
+.dems img {
+  width: 70%;
+}
+
+.grey {
+  background-color: #f4f5f6;
+}
+
+.btn-color {
+  background-color: #ff0500;
+  color: white;
+}
```

Comment:
Group with selector on line 74

Reply:
- [ ] You still didn't group this selector, as requested previously.